src/Shared/PackageManagers/PyPIProjectManager.cs

```diff
@@ -42,13 +43,50 @@ public class PyPIProjectManager : TypedManager<IManagerPackageVersionMetadata, P
         /// <inheritdoc />
         public override IEnumerable<ArtifactUri<PyPIArtifactType>> GetArtifactDownloadUris(PackageURL purl)
         {
-            string feedUrl = (purl.Qualifiers?["repository_url"] ?? ENV_PYPI_ENDPOINT).EnsureTrailingSlash();
+            Check.NotNull(nameof(purl.Version), purl.Version);
+            // TODO: Is this OK to do?
+            return this.GetArtifactDownloadUrisAsync(purl).ToEnumerable();
```

Comment:
@gfs I've never done anything like this before, unsure if there's a better solution.
One thought I had was just making GetArtifactDownloadUris async to begin with, but didn't want to do any contract breaking. 
Let me know what you think.

Reply:
I don't think I've used that APi before, but from the description - and since the tests seem to be functional, I think its fine.